File: src/main/java/ru/romanow/scc/warehouse/model/OrderItemResponse.java
package ru.romanow.scc.warehouse.model;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;
import lombok.experimental.Accessors;
import ru.romanow.scc.warehouse.domain.enums.OrderState;

import java.util.List;
import java.util.UUID;

@Data
@Accessors(chain = true)
@NoArgsConstructor
@AllArgsConstructor
public class OrderItemResponse {
    private UUID orderUid;
    private OrderState state;
    private List<ItemsShortInfo> items;
}

File: src/main/java/ru/romanow/warehouse/model/OrderItemResponse.kt
package ru.romanow.warehouse.model

import ru.romanow.warehouse.domain.enums.OrderState
import java.util.*

data class OrderItemResponse(
    val orderUid: UUID? = null,
    val state: OrderState? = null,
    val items: List<ItemsShortInfo>? = null
)